app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,12 @@
 <resources>
     <string name="app_name">ReadyForExams</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
I would suggest to name strings like this: name of the screen + purpose, for example if you create a title textview inside the landing screen I would call like landing_title_text or for the button text, landing_login_button_text . Remember, do not use short words and try to be consistent when naming things. I would also suggest to split strings by Fragments or Views with comments , something like this at the beginning of a screen strings: <!-- Landing Fragment -->